src/php/Compiler/CompilerFactoryInterface.php

```diff
@@ -4,24 +4,28 @@
 
 namespace Phel\Compiler;
 
-use Phel\Compiler\Analyzer\Environment\GlobalEnvironmentInterface;
+use Phel\Compiler\Compiler\EvalCompilerInterface;
+use Phel\Compiler\Compiler\FileCompilerInterface;
 use Phel\Compiler\Emitter\EmitterInterface;
 use Phel\Compiler\Emitter\OutputEmitterInterface;
 use Phel\Compiler\Lexer\LexerInterface;
 use Phel\Compiler\Parser\ParserInterface;
 use Phel\Compiler\Reader\ReaderInterface;
 
+/**
+ * deprecated Remove me without replacement. CompilerFacade is the one which should have the Interface!
```

Comment:
?

Reply:
Jap, I forgot to remove this one. Thanks for the reminder